frontend/public/components/resource-dropdown.tsx

```diff
@@ -89,31 +89,16 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
     return _.includes(selected, kind);
   };
   // Create dropdown items for each resource.
-  const items = resources.map((model) => (
-    <DropdownItem
-      key={referenceForModel(model)}
-      model={model}
-      showGroup={isDup(model.kind)}
-      checked={isKindSelected(referenceForModel(model))}
-    />
-  )) as OrderedMap<string, JSX.Element>;
-  // Add an "All" item to the top if `showAll`.
-  const allItems = (showAll
-    ? OrderedMap({
-        All: (
-          <>
-            <span className="co-resource-item">
-              <Checkbox id="all-resources" isChecked={isKindSelected('All')} />
-              <span className="co-resource-icon--fixed-width">
-                <ResourceIcon kind="All" />
-              </span>
-              <span className="co-resource-item__resource-name">All Resources</span>
-            </span>
-          </>
-        ),
-      }).concat(items)
-    : items
-  ).toJS() as { [s: string]: JSX.Element };
+  const items = resources
+    .map((model) => (
+      <DropdownItem
+        key={referenceForModel(model)}
+        model={model}
+        showGroup={isDup(model.kind)}
+        checked={isKindSelected(referenceForModel(model))}
+      />
+    ))
+    .toJS() as OrderedMap<string, JSX.Element>;
```

Comment:
This looks wrong... It's no longer an ordered map after calling `toJS()` right?

Reply:
I think you can just remove the `as`

```suggestion
    .toJS();
```